docs/root/configuration/listeners/listener_filters/proxy_protocol.rst

```diff
@@ -29,10 +33,35 @@ If there is a protocol error or an unsupported address family
 Statistics
 ----------
 
-This filter emits the following statistics:
+This filter emits the following general statistics, rooted at *downstream_cx_proxy_proto*
```

Comment:
I think they should be rooted at `proxy_proto` or possibly `downstream_proxy_proto`. `cx` should be included in the metric names if applicable (this is the convention everywhere).

Thinking about how they'll look if/when you add a `stat_prefix`, I think `proxy_proto.<my prefix>.v1.found` looks good.

Reply:
Sure. Let's use `downstream_proxy_proto` because I don't want these stats to be confused with the upstream proxy protocol transport socket.

SG on stat prefix. BTW it will be `downstream_proxy_proto.<my prefix>.versions.v1.found`. The `versions` part is following pre-existing style from other filters (ref RBAC, TLS upstream transport socket). It also makes the tag extractor more resilient to future stat additions.